docs/victory-area/docs.md

```diff
@@ -40,6 +40,16 @@ Use the `x` and `y` data accessor props to determine how the component defines d
 
 *examples:* `y="employees.salary"`, `y={["employees", "salary"]}`
 
+### sortKey
+
+Use the `sortKey` prop to indicate how data should be sorted. This prop is
+given directly to the lodash `sortBy` function to be executed on the final
+dataset: https://lodash.com/docs/4.17.4#sortBy
```

Comment:
As a markdown link, please. Maybe like
 
`This prop is given directly to [the lodash `sortBy` function]...` 

with the link at the bottom on the file like 
`[the lodash `sortBy` function]: https://lodash.com/docs/4.17.4#sortBy`